contracts/TopShot.cdc

```diff
@@ -654,10 +665,95 @@ pub contract TopShot: NonFungibleToken {
                         setID: self.data.setID,
                         numMomentsInEdition: TopShot.getNumMomentsInEdition(setID: self.data.setID, playID: self.data.playID)
                     )
+                case Type<MetadataViews.Editions>():
+                    let name = self.getEditionName()
+                    let max = TopShot.getNumMomentsInEdition(setID: self.data.setID, playID: self.data.playID) ?? 0;
+                    let editionInfo = MetadataViews.Edition(name: name, number: UInt64(self.data.serialNumber), max: UInt64(max))
+                    let editionList: [MetadataViews.Edition] = [editionInfo]
+                    return MetadataViews.Editions(
+                        editionList
+                    )
+                case Type<MetadataViews.Serial>():
+                    return MetadataViews.Serial(
+                        UInt64(self.data.serialNumber)
+                    )
+                case Type<MetadataViews.Royalties>():
+                    return MetadataViews.Royalties(
+                        royalties: [
+                            // No royalty for Moments yet defined for 3rd party marketplaces
```

Comment:
Aren't marketplaces having to give a 5% royalty just like TopShot is? With image data being added here, no royalty data could lead some marketplaces to being in trouble. Any idea if/when this might change?

Reply:
Yes - we do collect 5% royalties for offers and the official marketplace. Definitely planning to add that here.

Reply:
Just a note that not having this puts marketplaces in a sticky spot. Now we have to be aware of when something is a TopShot moment and treat its payout differently than others in terms of it's royalty configuration

Reply:
So topshot expects to have a royalty cut but it is not in here? Or is this empty because they have no cut on 3rd party markets?